REST_API_DOCS.md

```
@@ -0,0 +1,1861 @@
+      "id_field": "post_id",
+      "id": "post_999"
+    }
+  }
+}
+```
+
+---
+
+### 내가 쓴 게시글 목록
+
+**GET** `/posts` 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)